lib/fluent/plugin_helper/compat_parameters.rb

```diff
@@ -199,6 +213,12 @@ def compat_parameters_formatter(conf)
         # TODO: warn obsolete parameters if these are deprecated
         attr = compat_parameters_copy_to_subsection_attributes(conf, FORMATTER_PARAMS)
 
+        # TODO: make a utility method in TimeFormatter to handle these conversion
```

Comment:
`time_as_epoch` shouldn't be implemented in TimeFormatter, because it's now have `time_type` for unixtime, float (and string). It's just for compatibility for existing plugins.